database/sql/src/main/scala/cromwell/database/slick/MetadataSlickDatabase.scala

```diff
@@ -34,7 +34,8 @@ class MetadataSlickDatabase(originalDatabaseConfig: Config)
   override def addMetadataEntries(metadataEntries: Iterable[MetadataEntry])
                                  (implicit ec: ExecutionContext): Future[Unit] = {
     val action = DBIO.seq(metadataEntries.grouped(insertBatchSize).map(dataAccess.metadataEntries ++= _).toSeq:_*)
-    runAction(action)
+    // workaround for large object support
+    if (isPostgresql) runTransaction(action) else runAction(action)
```

Comment:
Today-I-Learned.

1. Please also leave a link pointing at: https://stackoverflow.com/questions/3164072/large-objects-may-not-be-used-in-auto-commit-mode#answer-3164352
2. I'm thinking this could be consolidated into a reusable `def runLobAction()` method that does this `if` statement, a sibling of `def runAction()` and `def runTransaction()`

Reply:
done